packages/extension-ui/src/Popup/Signing/Bytes.tsx

```diff
@@ -53,11 +55,24 @@ export default styled(Bytes)`
     text-overflow: ellipsis;
     vertical-align: middle;
     width: 100%;
+    padding: 0.15rem;
 
-    pre {
-      font-family: inherit;
-      font-size: 0.75rem;
-      margin: 0;
+    &.pre {
+      padding: 0px;
+      
+      div {
+        padding: 0.15rem;
+        font-family: inherit;
+        font-size: 0.75rem;
+        margin: 0;
+        white-space: pre;
+        overflow: auto;
+        max-height: calc(100vh - 480px);
+        min-height: ${theme.boxLineHeight};
+        border: 1px solid ${theme.boxBorderColor};
+        background: ${theme.boxBackground};
```

Comment:
used the boxBackground theme here.

Reply:
Ok, in that case, makes sense - not singular use.